lighty-modules/lighty-gnmi/lighty-gnmi-sb/README.md

```diff
@@ -1,6 +1,16 @@
 # gNMI south-bound module
-is south-bound lighty module which manages connection with gNMI target.
-This module implements functionality to make CRUD operations on gNMI target.
+is south-bound lighty module which manages connection with gNMI targets. This module implements functionality to make
+CRUD operations on multiple gNMI targets. Which make it easy to read and manipulate with data in gNMI devices.
+
+Lighty gNMI augment ODL network-topology model with [gnmi-topology](../../../lighty-models/lighty-gnmi-models/lighty-gnmi-topology-model/src/main/yang/gnmi-topology.yang)
+model. This allows us to connect the gNMI device by adding new node with desired [parameters](https://github.com/PANTHEONtech/lighty/blob/master/lighty-models/lighty-gnmi-models/lighty-gnmi-topology-model/src/main/yang/gnmi-topology.yang#L67) to gnmi-topology
```

Comment:
`allows us` -> `provides the option` / `provides the possibility`

Reply:
done